docs/README_standalone.md

```diff
@@ -16,44 +16,29 @@ The command `npx` is also able to install `angular-cli-ghpages` on the first usa
 
 ## Usage
 
-Execute `npx angular-cli-ghpages` in order to deploy the project with a build from `dist` folder.  
-**Note: You have to create the `dist` folder first (e.g. by running `ng build --prod`).**
+Execute `npx angular-cli-ghpages` in order to deploy the project with a build from `dist` folder. (`dist` is the default)
+**Note: You have to create the `dist` folder first (e.g. by running `ng build`).**
 
-Usage:
+Since Angular CLI 6 the build artifacts will be put in a subfolder, eg. `dist/PROJECTNAME`.
+Since Angular CLI 17 the build artifacts will be put in a subfolder, followed by the folder `browser`, eg. `dist/PROJECTNAME/browser`.
```

Comment:
```suggestion
Since Angular CLI 17 the build artifacts will be put in a subfolder, followed by the folder `browser`, e.g. `dist/PROJECTNAME/browser`.
```